scripts/tutorials/tutorials.lisp

```diff
@@ -8,17 +8,17 @@
 
 
```

Comment:
I think this ordering makes a lot of sense! But there's something else we'll need to change too:
https://github.com/evanbowman/skyland-beta/tree/master/scripts/tutorials/test

I wrote some test cases for tutorials that run during regression. If I had planned ahead better, I would have named the test case files with the name of the tutorial instead of the tutorial number. After rearranging the tutorial list, the test cases will be mismatched with the tutorials and will need to be renumbered.